### ISSUE TITLE
WCAG 3.1.1: Language of Page

### ISSUE THREAD

**Issue by arnfinn** [mong/mongts#1088]
Observations:

`<html>` element does not have a `[lang]` attribute

https://www.skde.no/pasientstrommer/

https://www.skde.no/helseatlas/en/

https://www.skde.no/kvalitetsregistre/alle/sykehus
